tests/fixtures/typescript/my-addon/src/components/template-only.ts

```diff
@@ -0,0 +1,3 @@
+import templateOnly from '@ember/component/template-only';
```

Comment:
this file is needed so that Glint knows what type the hbs file is.

In TS, we cannot have bare-hbs-file-components.